DOWNLOAD_SNAPSHOT_JAR.md

```diff
@@ -0,0 +1,23 @@
+# Instructions to download the latest snapshot version of the validator
+
+The following workflows produce artifacts that are stored for a period of 90 days:
+
+More particularly, [`test_pack_doc.yml`](../.github/workflows/test_package.yml) is responsible to automatically test and package the project.
+This workflow is triggered on each commit of each branch. 
+
+As a result, the latest snapshot version of the validator can be found under the list of artifacts generated when this workflow is executed on the `master` branch.
+The application jar is named as "gtfs-realtime-validator-lib-v-*-SNAPSHOT.jar".
+
+This also applies to the latest webapp snapshot version, for which the webapp jar is named as "gtfs-realtime-validator-webapp-v-*-SNAPSHOT.jar".
```

Comment:
The latest snapshot builds of the validator library and webapp can be found under the [list of artifacts for the master branch](https://github.com/MobilityData/gtfs-realtime-validator/actions/workflows/test_package.yml?query=branch%3Amaster):
* Library - `gtfs-realtime-validator-lib-v-*-SNAPSHOT.jar`
* Webapp - `gtfs-realtime-validator-webapp-v-*-SNAPSHOT.jar`

...where `*` is the branch and SHA1 commit of the build.